test/index.js

```diff
@@ -1,28 +1,11 @@
 const test = require('tape');
-const Eth = require('ethjs');
-
-const { INFURA_PROJECT_ID } = require('rc')('infura', {
-  // eslint-disable-next-line node/no-process-env
-  INFURA_PROJECT_ID: process.env.INFURA_PROJECT_ID,
-});
+const Ganache = require('ganache');
 
 const { MethodRegistry } = require('../dist');
 
-const provider = new Eth.HttpProvider(`https://mainnet.infura.io/v3/${INFURA_PROJECT_ID}`);
+const { provider } = Ganache.server();
```

Comment:
It looks like we're just testing the `parse` method in these tests; we're not testing the `lookup` method, which actually makes us of the provider. So, could we just use a fake provider object here? That way we don't even need Ganache.